Description: fix some spelling mistakes.
Author: William Vera <billy@billy.com.mx>
        Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2016-11-28
Index: netdiscover-0.3-pre-beta7/doc/netdiscover.8
===================================================================
--- netdiscover-0.3-pre-beta7.orig/doc/netdiscover.8
+++ netdiscover-0.3-pre-beta7/doc/netdiscover.8
@@ -61,7 +61,7 @@ wireless networks with low signal level.
 .TP
 \fB\-f\fR
 Enable fast mode scan. This will only scan for .1, .100 and .254 on
-each network. This mode is usefull while searching for ranges being
+each network. This mode is useful while searching for ranges being
 used. After you found such range you can make a specific range scan to
 find online boxes.
 
@@ -83,7 +83,7 @@ When using -P, continue program executio
 to capture ARP packets passively.
 
 .SH USAGE
-If passive mode (-p), scan list (-l) or scan range (-r) options arent
+If passive mode (-p), scan list (-l) or scan range (-r) options aren't
 enabled, netdiscover will scan for common lan addresses.
 .PP
 Screen control keys 
@@ -102,7 +102,7 @@ Scroll up (or up arrow)
 
 .TP
 \fBa\fR
-Show arp replys list
+Show arp replies list
 
 .TP
 \fBr\fR
@@ -114,7 +114,7 @@ Close help screen or end application
 
 .SH CONFIG FILES
 There are 2 configuration files that netdiscover will look for, each
-time it is executed, if file doesnt exist it will use default values.
+time it is executed, if file doesn't exist it will use default values.
 You can use the -d switch to disable reading and loading configuration
 files.
 
@@ -131,7 +131,7 @@ Example:\fR
 
 .TP
 \fB~/.netdiscover/fastips\fR
-List contaning the last octect of the ips to be scanned on each
+List containing the last octect of the ips to be scanned on each
 subnet, when using fast mode, by default (1,100,154).
 
 Example:\fR
